src/main/java/frc/robot/RobotContainer.java

```diff
@@ -188,6 +194,7 @@ private void configureBindings() {
         );
         //vision
         operatorControllerStartButton.toggleOnTrue(new ToggleLED(vision));
+        driverDPadUp.toggleOnTrue(new PlayerStationAutoAlign(drivetrainSubsystem, vision));
```

Comment:
I changed this from toggleOnTrue to a simple onTrue button detect type. We spoke about how having it being a toggle could lock Kai from moving until the command is cancelled.